### PR TITLE
Adding Lesson Program Governance Policy

### PR DESCRIPTION
**Pull request by lexnederbragt**
The Executive Council Program Committee had as one of its tasks this year to provide a Lesson Program Governance Policy. This Pull Request is the result of the Committee's work. It also incorporates comments and suggestions received from Core Team and others.

DO NOT MERGE, this Pull Request is to be voted upon at the next meeting of the Executive Council. Once approved, this PR can be merged.

### CODE REVIEW

File: topic_folders/governance/lesson-program-policy.md [hunk 1]
```diff
@@ -1,49 +1,74 @@
...
+
+* Each Lesson Program will have a Lesson Program Governance Committee.
+* Each Lesson Program Governance Committee will comply with the general [committee policy as described in the Carpentries Handbook](https://docs.carpentries.org/topic_folders/governance/committee-policy.html)
+* In addition, each Lesson Program Governance Committee:
+    * Maintains a minimum of 3 members to serve as on the committee and act as a point of contact with The Carpentries Executive Council and Core Team.
+    * Follows all general policies as outlined by The Carpentries, including, but not limited to, the [Code of Conduct](https://docs.carpentries.org/topic_folders/policies/index_coc.html).
+    * Determines the curriculum and core curriculum for (each of) the Lesson Program’s workshop(s), together with the corresponding [Curriculum Advisory Committee](https://docs.carpentries.org/topic_folders/lesson_development/lesson_development_roles.html#curriculum-advisory-committees) or by delegating curriculum decisions to the corresponding Curriculum Advisory Committee
```
Comment (sstevens2) [inline topic_folders/governance/lesson-program-policy.md:29]:
```suggestion
    * Determines the curriculum and core curriculum for (each of) the Lesson Program’s workshop(s), together with the corresponding [Curriculum Advisory Committee](https://docs.carpentries.org/topic_folders/lesson_development/lesson_development_roles.html#curriculum-advisory-committees) or by delegating curriculum decisions to the corresponding Curriculum Advisory Committee.
```

File: topic_folders/governance/lesson-program-policy.md [hunk 2]
```diff
@@ -1,49 +1,74 @@
...
+    * Decides if and when new lessons are added to or removed from the Lesson Program or delegate the decision of adding or removing a lesson to the corresponding Curriculum Advisory Committee.
+    * Decides if one or more Curriculum Advisory Committees are needed for their curricula and how they are structured
```
Comment (sstevens2) [inline topic_folders/governance/lesson-program-policy.md:31]:
```suggestion
    * Decides if one or more Curriculum Advisory Committees are needed for their curricula and how they are structured.
```

File: topic_folders/governance/lesson-program-policy.md [hunk 3]
```diff
@@ -1,49 +1,74 @@
...
+    * Decides how and how often they engage the Lesson Program community/communities
```
Comment (sstevens2) [inline topic_folders/governance/lesson-program-policy.md:32]:
```suggestion
    * Decides how and how often they engage the Lesson Program community/communities.
```

File: topic_folders/governance/lesson-program-policy.md [hunk 4]
```diff
@@ -1,49 +1,74 @@
...
+    * Decides between:
+        * seeking approval from the Executive Council to elect members to the Lesson Program Governance Committee
```
Comment (sstevens2) [inline topic_folders/governance/lesson-program-policy.md:34]:
```suggestion
        * seeking approval from the Executive Council to elect members to the Lesson Program Governance Committee.
```

File: topic_folders/governance/lesson-program-policy.md [hunk 5]
```diff
@@ -1,49 +1,74 @@
...
+        * appointing members based on an application process.
+    * Decides committee members term lengths.
+    * Decides on requesting additional Roles and Responsibilities beyond those described in this policy from the Executive Council.
+    * Works with The Carpentries Core Team to maintain documentation (including the lesson program website) with information about its goals, workshops and curricula.
+    * Coordinates with The Carpentries to release and publish lessons with permanent identifiers on a regular basis or delegates this responsibility to the corresponding Curriculum Advisory Committee.
+    * Maintains and develops lessons in accordance with Carpentries guidelines as outlined in The Carpentries Curriculum Development Handbook.
+* The Executive Council:
+    * Decides if a new Lesson Program can join The Carpentries as regulated by the [Bylaws](https://docs.carpentries.org/topic_folders/governance/bylaws.html#lesson-programs) and the [Lesson Program Policy](https://docs.carpentries.org/topic_folders/governance/lesson-program-policy.html)
```
Comment (sstevens2) [inline topic_folders/governance/lesson-program-policy.md:42]:
```suggestion
    * Decides if a new Lesson Program can join The Carpentries as regulated by the [Bylaws](https://docs.carpentries.org/topic_folders/governance/bylaws.html#lesson-programs) and the [Lesson Program Policy](https://docs.carpentries.org/topic_folders/governance/lesson-program-policy.html).
```

**Comment by sstevens2** [comment]
I think this looks good!  Thanks Lex for the last edits and putting together the PR

**Comment by sstevens2** [comment]
This has has been approved by the EC in the 2021-12-21 meeting.

**Comment by sstevens2** [comment]
@lexnederbragt Does this need more review or can we go ahead and merge?

**Comment by lexnederbragt** [comment]
Please merge!

**Pull request merged**